kibana-reports/server/plugin.ts

```diff
@@ -50,9 +51,11 @@ export class OpendistroKibanaReportsPlugin
       OpendistroKibanaReportsPluginStart
     > {
   private readonly logger: Logger;
+  private readonly semaphore: SemaphoreInterface;
 
   constructor(initializerContext: PluginInitializerContext) {
     this.logger = initializerContext.logger.get();
+    this.semaphore = withTimeout(new Semaphore(1), 30000, new Error('timeout'));
```

Comment:
I think 30 sec is too less a timeout. may be 180 seconds?

Reply:
changed to 180 seconds